style.css

```diff
@@ -0,0 +1,17 @@
+body {background-color: moccasin;}
+
+h1 {color: royalblue;margin: 20px; text-align: center;}
+
+{<img src="/Images/mjm.jpeg"
```

Comment:
Lines 5 and 6 are not valid CSS. Please remove these lines and format this file properly.

Reply:
I made the changes to the style.css. I also made a change to the index.html but forgot to note it in the commit message.